SU-SWS/saa_victoria_subtheme
# Adapt 476  big secondary button

# READY FOR REVIEW

# Summary
- enforces button styles for top red banner and in-page vivid red banner

# Review By (Date)
- 7/19

# Urgency
- important for random button configuration to ensure design consistency

# Steps to Test

1. pull down branch, clear cache
1. add regular button, secondary button and big button into a top banner and make sure they are all the same classic secondary button.
2. add regular button, secondary button and big button into a in-page red no-image banner and make sure they are all the same classic purple button.

# Affected Projects or Products
- na

# Associated Issues and/or People
- JIRA ticket 476 testing
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
